### DISCUSSION TITLE
Direct and reverse streams

### DISCUSSION THREAD

**Discussion by Alex-Kharkov-71** [blakeblackshear/frigate#9691] [Ask A Question]
Good day. I have no so powerful hardware, so I get reverse stream for detect. Is it possible, after motion detection take snapshots from direct stream for passing to CompreFace and DeepStack for recognition? If it possible, please advice how to do it.
Thank in advance. Sorry for my bad English.
Please, do not advice me to upgrade hardware. Near Kharkov. No work, no adequate delivery, and military action in progress.
Thank for understanding.

**Comment by NickM-27** [comment]
I'm not sure what you mean by direct, this is essentially how doubletake works already

**Comment by Vladd1980** [comment]
this means a stream 0 with full development (2560*1440)  and stream 1280*720
